src/common/UserObjectType.js

```diff
@@ -0,0 +1,6 @@
+export type UserObjectType = {
+  username: ?string,
+  email: ?string,
+  id: ?string,
+  profile: ?Object,
```

Comment:
Hmmm. I'm not sure. What would the usage scenario of this be? If the consuming code wants to extend the user object is should be handled through Accounts.onCreateUser http://docs.meteor.com/api/accounts-multi.html#AccountsServer-onCreateUser